lib/command.js

```diff
@@ -1340,6 +1341,19 @@ Expecting one of '${allowedValues.join("', '")}'`);
     });
   }
 
+  /**
+   * Display an error message if conflicting options are used together.
+   * Called after checking for help flags in leaf subcommand.
+   *
+   * @api private
+   */
+  _checkForConflictingOptions(cmd) {
```

Comment:
I don't think the `cmd` argument is necessary.

Reply:
Oops, left over from an earlier version of code. Thanks. I should have reread it!

Reply:
Fixed.